Cargo.toml

```diff
@@ -25,12 +25,12 @@ swc_ecmascript = { version = "=0.14.1", features = ["parser", "transforms", "uti
 regex = "1.4.1"
 once_cell = "1.5.2"
 derive_more = { version = "0.99.11", features = ["display"] }
+deno_core = "0.67.0"
```

Comment:
Yikes, @magurotuna this is not good, we can't have `deno_core` as a regular dep, it must be `dev-dependency`

Reply:
ok I switched it to anyhow instead.